### PR TITLE
added read me

### PR DESCRIPTION
**Pull request by justprateek**
Hii , its my first time on open source so i am just checking if everything works fine.  Please merge it and i will make further changes to it very soon

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+A Simple Catch the flag game
+
```
Comment (AymKh) [inline README.md:2]: you could insert a screenshot here (level 01)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Deployed here: 
+
+
+Getting Started
+These instructions will get a copy of the project up and running on your machine for development and testing purposes.
+
+Prerequisites
+To run this project locally, the follow tools need to be installed:
```
Comment (AymKh) [inline README.md:10]: Only Node is needed
the user can use "npm install" to get all the dependencies

File: README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Node.js
+Express.js
+ejs
+body-parser
+
+Installation:
```
Comment (AymKh) [inline README.md:16]: This section should be named Development instead of Installation

File: README.md (hunk 1 above)
Comment (AymKh) [reply README.md:2]: CTF = Capture The Flag

File: README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+2. Make sure you have npm installed
+3. Move into the project directory and install it's dependencies:
+    Node.js
+    Express.js
+    ejs
+    body-parser
+4. To start the application run the following command
+    npm start
+5. visit http://localhost:5000/
+
```
[the remark this answers is not in the document]
Comment (justprateek) [reply README.md:27]: Hey I have added some of the suggestions. Have a look

**Review by AymKh — APPROVED** [review]
👍Great !👏
I will just mke few changes later

**Pull request merged**